tests/cases/BankAccountsTest.php

```diff
@@ -1,20 +1,26 @@
 <?php
-namespace MangoPay\Tests;
-require_once 'base.php';
+
+namespace MangoPay\Tests\Cases;
+
+require_once __DIR__ . "/../../vendor/autoload.php";
```

Comment:
Not sure about that. Is it still required using the `bootstrap` argument in the phpunit file?